Core/Source/MVVM/View.swift

```diff
@@ -27,6 +27,10 @@ public protocol View: class {
     /// `viewModel` back to `nil`.
     func unbindFromViewModel()
 
+    /// Rebind is called on a view when it is going to be reused with a new `ViewModel`. The default implementation is
+    /// to call `unbindFromViewModel` and then `bindToViewModel` but views can override.
+    func rebindToViewModel(_ viewModel: ViewModel)
```

Comment:
How about `rebind(to viewModel: ViewModel)`? I know the other language in here is still Swift-2ish, but might as well use more modern naming.